jekyll-exporter.php

```diff
@@ -149,28 +274,43 @@ function get_posts() {
 	function convert_meta( $post ) {
 
 		$output = array(
-			'id'      => $post->ID,
+			'post_id'      => $post->ID,
 			'title'   => get_the_title( $post ),
 			'date'    => get_the_date( 'c', $post ),
+			'last_modified_at'    => get_the_modified_date( 'c', $post ),
 			'author'  => get_userdata( $post->post_author )->display_name,
 			'excerpt' => $post->post_excerpt,
 			'layout'  => get_post_type( $post ),
 			'guid'    => $post->guid,
+			'slug'   => $post->post_name,
 		);
 
 		// Preserve exact permalink, since Jekyll doesn't support redirection.
-		if ( 'page' !== $post->post_type ) {
+		$ignore_permalink = array(); // array('page')
+		if ( !in_array($post->post_type, $ignore_permalink) ) {
 			$output['permalink'] = str_replace( home_url(), '', get_permalink( $post ) );
 		}
 
 		// Convert traditional post_meta values, hide hidden values.
+		$ignore_list = array(
+			'_*',
+			'ocean_*',
+			'classic-editor-remember',
+			'osh_disable_topbar_sticky',
+			'osh_disable_header_sticky',
+			'osh_sticky_header_style',
+			'ampforwp-amp-on-off',
+		);
+
```

Comment:
Can these configuration options be user configurable rather than hard-coded to specific setups?

Reply:
Yes, done. I still have left this as default because even if it is specific to some theme and plugins it cannot be harmful if you don't use them and OceanWP is a common theme.